Makefile

```diff
@@ -201,10 +207,10 @@ lint: check-single-includes clint lualint _opt_pylint _opt_shlint
 # Generic pattern rules, allowing for `make build/bin/nvim` etc.
 # Does not work with "Unix Makefiles".
 ifeq ($(BUILD_TYPE),Ninja)
-build/%:
+build/%: phony_force
 	$(BUILD_CMD) -C build $(patsubst build/%,%,$@)
 
-$(DEPS_BUILD_DIR)/%:
+$(DEPS_BUILD_DIR)/%: phony_force
 	$(BUILD_CMD) -C $(DEPS_BUILD_DIR) $(patsubst $(DEPS_BUILD_DIR)/%,%,$@)
```

Comment:
Note that the "phony_force" below are not really necessary here, but make sense for any "%" rule, so I think it's ok to have them added via this also.